pkgs/development/tools/misc/coreboot-toolchain/update.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#!nix-shell --pure -i bash -p nix cacert git getopt
+#!nix-shell -I nixpkgs=./. --pure -i bash -p nix cacert git getopt
```

Comment:
Should this not be `nixpkgs=./../../../..` otherwise it's just the current directory. Or does this work because update scripts are always executed from the nixpkgs root?

Reply:
Why is that even needed?

Reply:
I've ran it from the top level of the repo as well. I can remove it, thought it would be nice to just pin to the current checkout of nixpkgs. Either way is fine with me

Reply:
Please remove. I don't want to force the user to use that. If anyone wants to run bleeding edge, they should define that in their local NixOS config.

Reply:
done!